third_party/2and3/click/testing.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import (IO, Any, BinaryIO, ContextManager, Dict, Iterable, List,
+                    Mapping, Optional, Tuple, Union)
+
+from .core import BaseCommand
+
+clickpkg: Any
+
+class EchoingStdin:
+    def __init__(self, input: BinaryIO, output: BinaryIO) -> None: ...
+    def __getattr__(self, x: str) -> Any: ...
+    def read(self, n: int = ...) -> bytes: ...
+    def readline(self, n: int = ...) -> bytes: ...
+    def readlines(self) -> List[bytes]: ...
+    def __iter__(self) -> Iterable[int]: ...
```

Comment:
Shouldn't this return an iterable of `bytes`?

Reply:
Good catch, I mixed it with `bytes.__iter__` which has this type.